### PR TITLE
App Service & traffic manager samples

### PR DESCRIPTION
**Pull request by jianghaolu**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as a link to the swagger spec, used to generate the code.
- [ ] The `project.json` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: Samples/appservice.playlist [hunk 1]
```diff
@@ -0,0 +1 @@
+<Playlist Version="1.0"><Add Test="Samples.Tests.AppService.ManageLinuxWebAppWithTrafficManagerTest" /><Add Test="Samples.Tests.AppService.ManageLinuxWebAppBasicTest" /><Add Test="Samples.Tests.AppService.ManageLinuxWebAppStorageAccountConnectionTest" /><Add Test="Samples.Tests.AppService.ManageLinuxWebAppSqlConnectionTest" /><Add Test="Samples.Tests.AppService.ManageLinuxWebAppSourceControlTest" /><Add Test="Samples.Tests.AppService.ManageLinuxWebAppWithDomainSslTest" /></Playlist>
```
Comment (hovsepm) [inline Samples/appservice.playlist]: Please remove this file from PR

File: src/ResourceManagement/AppService/Microsoft.Azure.Management.AppService.Fluent/FunctionAppImpl.cs [hunk 2]
```diff
@@ -157,5 +164,138 @@ public FunctionAppImpl WithLatestRuntimeVersion()
...
+        {
+            return GetMasterKeyAsync().ConfigureAwait(false).GetAwaiter().GetResult();
+        }
+
+        public async Task<string> GetMasterKeyAsync(CancellationToken cancellationToken = default(CancellationToken))
+        {
+            string apiVersion = "2016-08-01";
+            // Tracing
+            bool _shouldTrace = ServiceClientTracing.IsEnabled;
+            string _invocationId = null;
```
Comment (hovsepm) [inline src/ResourceManagement/AppService/Microsoft.Azure.Management.AppService.Fluent/FunctionAppImpl.cs]: Could you add comments why we have this generated snippet here and we do not call underlying generated client method?

Comment (jianghaolu) [inline src/ResourceManagement/AppService/Microsoft.Azure.Management.AppService.Fluent/FunctionAppImpl.cs]: There's not generated method for this. This was missing from Swagger.

File: Samples/appservice.playlist (hunk 1 above)
Comment (jianghaolu) [inline Samples/appservice.playlist]: done.

File: src/ResourceManagement/AppService/Microsoft.Azure.Management.AppService.Fluent/FunctionAppImpl.cs (hunk 2 above)
Comment (jianghaolu) [inline src/ResourceManagement/AppService/Microsoft.Azure.Management.AppService.Fluent/FunctionAppImpl.cs]: Comments added.

**Comment by hovsepm** [comment]
@jianghaolu please resolve merge conflicts

**Pull request merged**